.github/workflows/bench.yml

```diff
@@ -155,9 +155,9 @@ jobs:
           sed 's/`//g' steps.md |\
             awk '!/^\| Command/ || !c++' |\
             awk '!/^\|:/ || !c++' |\
```

Comment:
Is this `sed -e '/^\|:/,$ { 1d; }` ?

Reply:
Possibly? I tried various things but `sed` not being really cross-platform made me switch to `awk`.